README.md

```diff
@@ -1 +1,54 @@
-# hello-rails-back-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Rails Back End
+
+## Front-End project
+[Link](https://github.com/leopiresgoss/hello-react-front-end/)
```

Comment:
Nice job 👍  with the app all thing work fine. Please as you add link for the Front-End repo here in the Back-End `README.md` file do the same thing by adding link to the Back-End repo in the Front-End `README.md` file.